docs/guides/03-Styling-your-visualization-using-Expressions.md

```diff
@@ -0,0 +1,198 @@
+- Using the `filter` property. The filter property multiplies the color alpha channel (both fill and stroke) of the features with its value. This, however, has a special semantic regarding viewport aggregation expressions. We'll look at this later.
+
+
+### Numeric and boolean expressions basics
+
+In CARTO VL numeric expressions are a first class citizen. You have access to math like `+`, `-`, `*`,`/`,`^`,`log`,`sin`... out of the box.
+
+You’ll also be able to get boolean-like values within CARTO VL. Some ways to get these “boolean-like” types are comparison functions (`==`, `!=`, `>`, `<`, `>=`), utils (`between`, `in`) and logic functions (`not`, `or`, `and`).
+
+We talk about boolean-like types and not just boolean types because there is no “boolean” type in CARTO VL. Every “boolean-like” type is actually emulated by the numeric type.
+This makes our boolean logic a [fuzzy boolean logic](https://en.wikipedia.org/wiki/Fuzzy_logic), making most animations (blendings) work out of the box.
+
+### Using dataset feature properties
+To refer to a dataset feature property we can use the dollar sign. For example, `$price` refers to the features property named ‘price’ in the dataset.
+
+We’ll use this extensively in the Data-driven visualization guide.
+
+### Accessing and modifying visualization properties
+
+Once a visualization has been created, you can access and modify all the styling properties through the returned [Visualization]().
+
+Once you have a `Visualization` object, you can:
+- Get its property values by using the `.value` getter: `const currentColor = viz.color.value;`
+- Modify its property values by using the`.blendTo()` method: `viz.color.blendTo(‘green’);`
+
+<div class="example-map">
+    <iframe
+        id="population-density-basic"
+        src="/developers/carto-vl/examples/maps/guides/viz-properties/accessing-viz-properties.html"
+        width="100%"
+        height="500"
+        style="margin: 20px auto !important"
+        frameBorder="0">
+    </iframe>
+    <a href="/developers/carto-vl/examples#example-color-spaces">View my source code!</a>
+</div>
+
+
+### The String and the JS Visualization APIs
+
+CARTO VL provides two equivalent and similar APIs to define visualizations. Until now, we have seen the String API which is nothing more than a small layer of syntax sugar for the JS API. In fact, internally, the usage of the String API is implemented through the usage of the public JS API.
+
+Usage of the JS API can facilitate some advanced topics, but both are equivalent and everything that can be done through the JS API can be done with the String API and vice versa.
+
+In the reference documentation you’ll be able to see examples with both APIs for each expression, but let’s see some examples:
+
+```Javascript
+// Usage of the String API
```

Comment:
I would try to get some space between the two code blocks. One option is to extract the comment and use it as a little previous title

Reply:
:+1: